src/common/keyboard_layout.cpp

```diff
@@ -44,6 +44,10 @@ std::wstring LayoutMap::LayoutMapImpl::GetKeyName(DWORD key)
     {
         result = it->second;
     }
+    if (key == 0)
+    {
+        result = L"None";
+    }
```

Comment:
I think this isn't required, since there is already a key entry for None in the list

Reply:
Since None is somewhat added after the fact in the existing code here:
https://github.com/microsoft/PowerToys/blob/f5a256fb352d3d6028939546a961504434556d0e/src/common/keyboard_layout.cpp#L317-L319
without these lines, the key mapping is listed as undefined.
![image](https://user-images.githubusercontent.com/49463671/93657945-a5fb3500-f9fc-11ea-9913-5c9d2bb315ae.png)
This is what happens with these lines removed.